packages/odata-v2/src/request-builder/response-transformers.ts

```diff
@@ -4,13 +4,25 @@ import { DeSerializers } from '../de-serializers/de-serializers';
 import { Entity } from '../entity';
 import { getSingleResult, getCollectionResult } from './response-data-accessor';
 
+/**
+ * Convert the payload of the OData response to undefined.
```

Comment:
[pp] Maybe it would be nice to add a bit more context what this is good for. Looks a bit weird without context.